src/eba_handlers.eliom

```diff
@@ -112,10 +118,10 @@ let sign_up_handler' () email =
 let forgot_password_handler service () email =
   try_lwt
     lwt userid = Eba_user.userid_of_email email in
-    let text = "Hi,\r\nTo set a new password, please click on this link: " in
-    let act_key = generate_act_key ~service ~text email in
-    Eliom_reference.Volatile.set Eba_msg.activation_key_created true;
-    Eba_user.add_activationkey ~act_key userid
```

Comment:
I haven't run the code. It is nice that you "de-inlined" the version of `send_act` that was hiding inside `forgot_password_handler`, but the real fix is removing the call to `Eba_user.add_activationkey` (which is an orthogonal improvement), right?